### ISSUE TITLE
kubeadm init for the master?

### ISSUE THREAD

**Issue by luxas** [apprenda/kubernetes-ovn-heterogeneous-cluster#19]
It would be super-cool if I could just `kubeadm init` on the master and then install some extra things on top when initializing the master :)

@pires Would that be easily achieveable?

**Comment by pires** [comment]
The problem is not the master but the worker nodes. However, it would be super cool to have it working fully for Linux and Windows.

**Comment by luxas** [comment]
Yes, but couldn't we replace 80-90% of the master init steps with `kubeadm` to minimize duplication.
The only project-specifc tasks I saw for the master init was the OVS stuff, which kubeadm doesn't handle in any case...

I know it won't work OOTB for the nodes initially, but doing it in steps; first for the master, then minimizing the delta for the linux nodes and then checking what's left for the windows nodes, we could probably make it eventually. WDYT?

**Comment by pires** [comment]
I think it's a great idea. I just won't be able to work on that, for the time being.
